strr-strata-web/app/pages/strata-hotel/dashboard/index.vue

```diff
@@ -69,8 +70,35 @@ const { data: strataHotelList, status } = await useAsyncData(
   }
 )
 
+function isDraft (status: string) {
+  return status === 'Draft'
+}
+
+const deleting = ref(false)
+async function deleteDraft (row: any) {
+  try {
+    deleting.value = true
+    row.class = 'bg-red-50 animate-pulse'
+    row.disabled = true
+    await Promise.all([
+      new Promise(resolve => setTimeout(resolve, 500)),
```

Comment:
Is this also a small delay for animation?

Reply:
yes exactly